Rakefile

```diff
@@ -87,7 +87,15 @@ namespace :dependencies do
     task :apply do
       next unless Dir.exist?(File.join(Dir.home, '.mobile-secrets/.git')) || ENV.key?('CONFIGURE_ENCRYPTION_KEY')
 
-      sh('FASTLANE_SKIP_UPDATE_CHECK=1 FASTLANE_ENV_PRINTER=1 bundle exec fastlane run configure_apply force:true')
+      command_compontents = %w[
+FASTLANE_SKIP_UPDATE_CHECK=1
```

Comment:
I wonder why my editor didn't warn me about this 🤔

Reply:
> I wonder why my editor didn't warn me about this 🤔

Because I use Vim with [syntastic](https://github.com/vim-syntastic/syntastic) and it can't use RuboCop via Bundler, it expects to be available as a system gem.